fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/dao/api/ResourceDAO.java

```diff
@@ -84,6 +84,20 @@ int historyCount(String resourceType, String logicalId, Timestamp fromDateTime)
      * @throws FHIRPersistenceDBConnectException
      */
     List<Resource> search(SqlQueryData queryData) throws FHIRPersistenceDataAccessException, FHIRPersistenceDBConnectException;
+
+    /**
+     * Executes the search contained in the passed SqlQueryData, using it's encapsulated search string and bind variables.
+     * This function is used by _include search only
```

Comment:
I suggest removing this now that we have the @implNote

Reply:
done